tests/integration/set_tblproperties/test_set_tblproperties.py

```diff
@@ -10,28 +12,39 @@ def schema(self):
     def models(self):
         return "models"
 
+    def check_tblproperties(self, model_name: str, properties: List[str]):
+        results = self.run_sql(
+            "show tblproperties {database_schema}.{model_name}",
+            fetch="all",
+            kwargs=dict(model_name=model_name),
+        )
+        tblproperties = [result[0] for result in results]
+
+        for prop in properties:
+            assert prop in tblproperties
+
+    def check_snapshot_results(self, num_rows: int):
+        results = self.run_sql("select * from {database_schema}.my_snapshot", fetch="all")
+        self.assertEqual(len(results), num_rows)
+
     def test_set_tblproperties(self):
         self.run_dbt(["seed"])
         self.run_dbt(["run"])
         self.run_dbt(["run"])
+        self.run_dbt(["snapshot"])
+        self.run_dbt(["snapshot"])
```

Comment:
Why do we need to run it twice? For idempotency check?

Reply:
Yes, just checking it doesn't break the second run.